Hashlink.js

```diff
@@ -72,12 +130,12 @@ export class Cipher {
    * mechanisms such as new cryptographic hashing, base-encoding, and
    * resolution mechanisms.
    *
-   * @param {string} algorithm - An algorithm identifier.
-   * @param {Function} transform - A Function(input) that transforms input
-   *   data to output data.
+   * @param {Transform} transform - A Transform class that has a .encode()
+   *   and a .decode() method. It must also have an ```identifier``` and
+   *   ```algorithm``` property.
```

Comment:
```suggestion
   *   `algorithm` property.
```

Reply:
Done in ebd661c1fcaceff5eccd5269caa818ff134c9af5.